_posts/2023-02-28-admin-panels-for-index-operations.md

```diff
@@ -0,0 +1,138 @@
+
+![Image: Shrink-started]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-started.jpg){:.img-fluid }
+
+You will see a notification showing the shrink operation has started successfully. After the shrink operation is completed (this can be anywhere from minutes to hours), a new notification will be shown saying that the source index has been successfully shrunken.
+
+![Image: Shrink-completed]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-completed.jpg){:.img-fluid }
+
+### Open and close index operation
+
+You can select multiple indices, except for the backing indices of a data stream, to open or close.
+
+If you don't need to read or search old indices, but you don't want to delete them, you can use close operation to     close indices which will maintain the data but will  then have small overhead on the cluster. Additionally, when you want to add a new analyzer to an existing index you must close the index, define the analyzer, then open the index. A closed index is blocked for read and write operations so you must type the word close to confirm your action:
+
+![Image: Close-index]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/close-index.jpg){:.img-fluid }
+
+Opening an index is easier as you can select multiple indices to open even though some indices are already open:
+
+![Image: Open-index]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/open-index.jpg){:.img-fluid }
+
+### Split index
+
+When users choose to split an index the status of that index is checked. If the index not able to be split, actions the user can take to make the index ready for splitting are provided. A list of shard numbers the index can be split into are provided so that the user does not need to calculate it manually.
+Users can specify the number of replicas and then associate the new index with an existing index or use a new alias. They can also use the JSON editor to specify any index settings as well.
+
+![Image: Split page]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/splitindex-normal.jpg){:.img-fluid }
+![Image: Split page]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/splitindex-notready.jpg){:.img-fluid }
+
+## In conclusion
+
+**Try it**
+
+To try out the new index management features, check out these features on the [official playground](https://playground.opensearch.org/app/opensearch_index_management_dashboards#/indices).
+
+## Next steps
+
+We’ll be releasing more Index Management UI features related to data streams, monitor metrics and so on. If you have any advice on what we currently have or will release, please leave a message on the [OpenSearch forum](https://forum.opensearch.org/).
```

Comment:
Consider referencing OpenSearch 2.5 in here and linking back to the blog: https://opensearch.org/blog/opensearch-2-5-is-live/ since it mentions the index management enhancements.

Reply:
Done for this.